### ISSUE TITLE
DUO110: false positive on re.compile

### ISSUE THREAD

**Issue by scorpp** [duo-labs/dlint#16]
The following code triggers `DUO110` which is regarding `compile` built-in, not the `re.compile`

```
from re import compile, search

class CustomValidator(object):
    digits = compile('[0-9]+')
```

**Comment by mschwager** [comment]
Good catch!

Tracking my thoughts here:

- There may be some overlap here with improving `BadModuleAttributeUseLinter` (#1)
- We should improve `BadBuiltinUseLinter` such that it can detect if a builtin function has been overwritten by an import

Thanks!

**Comment by mschwager** [comment]
@scorpp,

This false positive will be gone in the next release of Dlint :)
